package.json

```diff
@@ -6,6 +6,7 @@
     "node": ">=8.9.0"
   },
   "main": "dist/raw.macro.js",
+  "types": "raw.macro.d.ts",
   "license": "MIT",
   "files": [
     "dist/"
```

Comment:
You'll need to add the file to this property as well

Reply:
Thank you for guidance! By the way I don't know why other lines in my later commit were prettier-ed.

Reply:
nah it's fine